projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -4294,7 +4294,7 @@ public function render_radio_field( $id, $label, $value, $class, $required, $req
 		$field = $this->render_label( '', $id, $label, $required, $required_field_text );
 		foreach ( (array) $this->get_attribute( 'options' ) as $option_index => $option ) {
 			$option = Grunion_Contact_Form_Plugin::strip_tags( $option );
-			if ( $option ) {
+			if ( is_string( $option ) ) {
```

Comment:
I wonder if we should add a `&& $option !== ''` in here (and below). Do we want to start allowing empty-string as an option too, or just zero?

Reply:
I was thinking about that. The reason I didn't add that is that I think we shouldn't allow saving options like that if we don't want to render them. Right now nothing seems to stop users from saving stuff like this:
<img width="350" alt="Screen Shot on 2022-07-01 at 14-32-11" src="https://user-images.githubusercontent.com/1242807/176895090-1c8bb214-333e-4995-b58c-78756e778a21.png">

Reply:
Changed in https://github.com/Automattic/jetpack/pull/24926/commits/d178061864752a18fdd136e55c85b2851cfca5ba